react-graphql-apollo-tutorial.md

```diff
@@ -1145,26 +1147,22 @@ const RepositoryItem = ({ ... }) => (
+* Implement other mutations in the RepositoryItem component
+  * Implement the `removeStar` mutation when the `viewerHasStarred` boolean is true
+  * Show a button with the watchers count which should be used to watch/unwatch a repository
+    * Implement the `updateSubscription` mutation from GitHub's GraphQL API to watch/unwatch a repository based on the `viewerSubscription` status
+* Invest three minutes of your time and take the {{% a_blank "quiz" "https://www.surveymonkey.com/r/5GJQWXC" %}}
 
 {{% chapter_header "GraphQL Query/Mutation with Higher-Order Components in React" "react-apollo-client-query-mutation-higher-order-component" %}}
 
-In the previous section, you have introduced the Query and Mutation components from React Apollo to connect your data-layer (Apollo Client) with your view-layer (React). Whereas the Query component executes the query when it is rendered, the Mutation component gives access to a function that can be used to trigger the mutation. Both components use the render props pattern to make the results accessible in their child functions.
-
-There exists a widely accepted alternative to React's render prop pattern: [Higher-Order Components (HOC)](https://www.robinwieruch.de/gentle-introduction-higher-order-components/). The React Apollo package implements a Higher-Order Component for queries and mutations as well. However, the team behind Apollo doesn't advertise it and decided in favor of render props as their first class citizen. Nonetheless, this section shows you the alternative by using a Higher-Order Component instead of a Render Prop although the tutorial will continue to use the render prop pattern afterward. So you are free to try out the alternative, but you may want to stick to render props afterward for the sake of this tutorial.
+We've done Query and Mutation components from React Apollo to connect a data-layer (Apollo Client) with a view-layer (React). The Query component executes the query when it is rendered, whereas the Mutation component gives access to a function that triggers the mutation. Both components use the render props pattern to make the results accessible in their child functions.
 
-First, simply imagine you would have already access to the query result in the Profile component's arguments. There is no Query component needed in the component itself.
+[Higher-Order Components (HOC)](https://www.robinwieruch.de/gentle-introduction-higher-order-components/) is a widely accepted alternative to React's render prop pattern. The React Apollo package implements a Higher-Order Component for queries and mutations as well, though the team behind Apollo doesn't advertise it, and even spoke in favor of render props as their first choice. Nonetheless, this section shows you the alternative, using a Higher-Order Component instead of a Render Prop, though the tutorial will continue to use the render prop pattern afterward. There is no Query component needed in the component itself:
```

Comment:
"There is no Query component needed in the component itself:" by itself makes no sense. The first sentence adds context to it: "... imagine you would have already access to the query result in the Profile component's arguments"

Reply:
Clarification: It should maybe say: "Let's imagine you would have already access to the query result in the Profile component's arguments. Then there is no Query component needed in the component itself:"

Reply:
I am assuming you mean "Lets imagine you already have access to the query result..."

Reply:
"If you already have access to the query result in the Profile component's arguments, there is no Query component needed in the component itself:"

Reply:
That's good, yeah!